### DISCUSSION TITLE
How to prevent 2 dynamic rigid bodies from pushing each other?

### DISCUSSION THREAD

**Discussion by TheMetaverseEngineer** [dimforge/rapier#667] [Q&A]
Hi, I want to know how to prevent 2 rigid bodies from pushing each other. I read about the dominance group in the documentation but that only works for only 1 rigid body. imagine if the 2 rigid bodies are moving toward each other and none can push the other one. is there any solution?

**Comment by nstoddard** [comment]
Colliders are responsible for detecting collisions between objects, not rigid bodies. Take a look at [collision groups](https://rapier.rs/docs/user_guides/rust/colliders#collision-groups-and-solver-groups); that should be able to do what you need. For instance, you could set `membership` and `filter` for the first object to group 1, and set them to group 2 for the second object; then they won't collide with each other and will pass through each other instead.
